paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityLeftHanded.java

```diff
@@ -0,0 +1,44 @@
+package com.denizenscript.denizen.paper.properties;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.properties.entity.EntityProperty;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.Mob;
+
+public class EntityLeftHanded extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name left_handed
+    // @input ElementTag(Boolean)
+    // @plugin Paper
+    // @description
+    // Controls whether the mob is left-handed or not.

```

Comment:
Can probably be just `Whether a mob is left-handed`, and maybe something about it being randomly set for each mob (iirc?)